### ISSUE TITLE
 Could not find ::. Required by:  project :app

### ISSUE THREAD

**Issue by enriqueFcoG** [mauron85/react-native-background-geolocation#569]
## Your Environment

"@mauron85/react-native-background-geolocation": "^0.6.3"
"react": "17.0.2",
"node": "17.8.0"
java 17.0.2

* Plugin version:
* Platform: Android
* OS version: macOS Monterey M1 2020
* Device manufacturer and model:

* Running in Simulator: Yess 
* React Native version: 0.68.2
* Plugin configuration options: 
* Link to your project:

## Context

i upgrade my project from react native 0.62.2 to 0.68.2 and I got. then I run my app on android emulator and I got this error 

`Could not determine the dependencies of task ':app:mergeDebugAssets'.
> Could not resolve all task dependencies for configuration ':app:debugRuntimeClasspath'.
   > Could not find ::.
     Required by:
         project :app`

## Steps to Reproduce

1. npm install @mauron85/react-native-background-geolocation 
2. manual setup
3. run app on android device
4.

**Comment by luismz95** [comment]
same issue.

**Comment by Alipatel05** [comment]
Same issue I think the library hasn't been updated since long and that is causing problems

**Comment by enriqueFcoG** [comment]
Yes, i had to change my react native versión to 0.67.1 and it works fine

**Comment by enriqueFcoG** [comment]
> Same issue I think the library hasn't been updated since long and that is causing problems 
> 
> 

Yes, i had to change my react native versión to 0.67.1 and it works fine

**Comment by Salmankhan033** [comment]
Same issues.
Is there any updated library or methods that how to get the locations in background mode ?

**Comment by chanphiromsok** [comment]
> Same issues.
> 
> Is there any updated library or methods that how to get the locations in background mode ?

might need commercial library call react-native-background-geolocation

**Comment by Linuhusainnk** [comment]
> > Same issue I think the library hasn't been updated since long and that is causing problems
> 
> Yes, i had to change my react native versión to 0.67.1 and it works fine

What about SDK versions?

**Comment by zavex** [comment]
Same error with:
` "react-native": "0.74.5",
`

```
> Configure project :react-native-reanimated
Android gradle plugin: 8.2.1
Gradle: 8.6

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:debugCompileClasspath'.
   > Could not find ::.
     Required by:
         project :app

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

```
